mms/tests/unit_tests/test_validate_messages.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+from mms.mxnet_model_service_error import MMSError
+from mms.utils.validators.validate_messages import ModelWorkerMessageValidators
+from mms.utils.model_server_error_codes import ModelServerErrorCodes
+import json
+
+
+def get_jsonified_str(string):
```

Comment:
Why we need this function? I think json.loads() is more readable and shorter.

Reply:
Done. ad207eba2c5195b6f238faf41f4231369ca9e969
Removed it altogether